docs/Project.toml

```diff
@@ -8,6 +8,6 @@ Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
 Symbolics = "0c5d862f-8b57-4792-8d23-62f2024744c7"
 
 [compat]
-Documenter = "0.25"
+Documenter = "0.27"
 MAT = "0.10"
-Symbolics = "0.1.24"
+Symbolics = "1.4"
```

Comment:
not 100% sure but i think this requires that we update lazysets..

Reply:
Maybe in general but I ran it locally with this version and it worked.